src/main/java/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -1339,38 +1339,27 @@ public static int getUnitPowerForSorting(final Unit current, final boolean defen
       final Collection<TerritoryEffect> territoryEffects) {
     final boolean lhtrBombers = games.strategy.triplea.Properties.getLHTR_Heavy_Bombers(data);
     final UnitAttachment ua = UnitAttachment.get(current.getType());
-    final int rolls;
-    if (defending) {
-      rolls = ua.getDefenseRolls(current.getOwner());
-    } else {
-      rolls = ua.getAttackRolls(current.getOwner());
-    }
+    final int rolls = (defending) ?  ua.getDefenseRolls(current.getOwner()) :
```

Comment:
Indeed. Adding them is to  make the ternary operator more explicit and is a bit of a stylistic thing. The non-paren case I found less readable.  
Ternary operators can be nice, the optional parens I believe are often recommended to keep the conditional part of the syntax a bit more explicit.